File: app_mvp/src/main/java/com/example/jqorz/test1/mvp/delegate/fragment/FragmentMvpDelegateCallback.java
package com.example.jqorz.test1.mvp.delegate.fragment;


import com.example.jqorz.test1.mvp.MvpView;
import com.example.jqorz.test1.mvp.delegate.BaseDelegateCallback;
import com.example.jqorz.test1.mvp.presenters.MvpPresenter;

/**
 * Created by jqorz on 2017/8/28.
 * 生命周期回调
 */

public interface FragmentMvpDelegateCallback<P extends MvpPresenter, V extends MvpView>
        extends BaseDelegateCallback<P, V> {

}

File: app_mvp/src/main/java/com/example/jqorz/mvptest/lib_kotlin/delegate/fragment/FragmentMvpDelegateCallback.kt
package com.jqorz.tallybook.base.mvp.delegate.fragment

import com.jqorz.tallybook.base.mvp.MvpView
import com.jqorz.tallybook.base.mvp.delegate.BaseDelegateCallback
import com.jqorz.tallybook.base.mvp.presenters.MvpPresenter

/**
 * 生命周期回调
 */
interface FragmentMvpDelegateCallback<P : MvpPresenter<V>?, V : MvpView> : BaseDelegateCallback<P, V>